src/components/TrackPain/NewPainEntry.js

```diff
@@ -6,7 +6,7 @@ import Typography from "@material-ui/core/Typography";
 import Grid from "@material-ui/core/Grid";
 import Slider from "@material-ui/lab/Slider";
 import DateFnsUtils from "material-ui-pickers/utils/date-fns-utils";
-import { MuiPickersUtilsProvider } from 'material-ui-pickers';
+import MuiPickersUtilsProvider from "material-ui-pickers/MuiPickersUtilsProvider";
```

Comment:
I suggest to do this import to solve the conflict: 
```
import { MuiPickersUtilsProvider } from "material-ui-pickers";
import { DateTimePicker } from "material-ui-pickers";
```
I have tested it in Chrome, IE 9, 10, 11 and Edge. It works.